MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,47 @@
+# Migration guide
+
+## From 0.4.0 to 0.5.0
+
+**Step 1**: First of all you need to make sure you upgraded Steiger package to version 0.5.0 or higher.
+
+**Step 2**: You need to install `@feature-sliced/steiger-plugin` package (that contains all FSD rules to run checks in your project). Run one of the following commands based on the package manager you use.
+
+**pnpm**:
+
+```shell
+pnpm add @feature-sliced/steiger-plugin
```

Comment:
suggestion: I think this package should be installed to dev dependencies

Reply:
100%, missed that thing